modules/nf-core/metabuli/build/main.nf

```diff
@@ -0,0 +1,57 @@
+process METABULI_BUILD {
+    tag "$meta.id"
+    label 'process_medium'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metabuli:1.1.0--pl5321hd6d6fdc_0':
+        'biocontainers/metabuli:1.1.0--pl5321hd6d6fdc_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path taxonomy_names, stageAs: 'taxonomy/names.dmp'
+    path taxonomy_nodes, stageAs: 'taxonomy/nodes.dmp'
+    path taxonomy_merged, stageAs: 'taxonomy/merged.dmp'
+    path accession2taxid, stageAs: 'taxonomy/*'
+
+    output:
+    tuple val(meta), path("$prefix"), emit: db
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    make_merged = taxonomy_merged ? "" : "touch taxonomy/merged.dmp"
+    """
+    $make_merged
+    realpath $fasta > fasta.txt
```

Comment:
```suggestion
    realpath -s $fasta > fasta.txt
```

I think this would be safer in the context of Nextflow staging, which relies heavily on symlinks, and thus will point to the files as inside the working directory